src/main/resources/application.properties

```diff
@@ -100,4 +100,6 @@ logging.level.edu.hawaii.its.api=DEBUG
 logging.level.org.springframework=WARN
 
 # =========================================================================
-app.environment=dev
\ No newline at end of file
+app.environment=dev
+
+grouping.api.server.type=REAL
```

Comment:
I think this line should be at the top of `application.properties` for how important it is. I was also thinking we should name the property value GROUPER instead of REAL to be more specific.